### PR TITLE
Added attachable process

### LINKED ISSUE #316: [Feature]: Server Debugger
### What feature?

Make server side debuggable

### PR DESCRIPTION
**Pull request by Njima1572**
## Which Issue was this PR made for?

- Closes #316

## What is within the scope with this PR

- Make server side debuggable with debugger

## What is NOT in the scope with this PR

- Actual debugging

### CODE REVIEW

**Comment by Njima1572** [comment]
AUTOMATED: Deployed to https://pr-317.dev.mochatto.com

File: README.md [hunk 1]
```diff
@@ -25,6 +25,15 @@ Mochatto is an open-source proximity voice chat application for virtual meetings
 1. `make dev`
 1. `make dev-up`
 
+
+#### Debugging Server
+
+Requires VSCode
+1. `make dev-up`
+2. open `mochatto/server` in VSCode
+3. Go to Debug tab
```
Comment (siberowl) [inline README.md:34]: I'm not sure if VSCode changed but for me it's:
Rub -> Start Debugging -> Docker: Attach to node (it should automatically be attached)
nbd though

File: docker-compose.yaml [hunk 2]
```diff
@@ -8,6 +8,7 @@ services:
       target: base
     ports:
       - '4000:4000'
+      - '9229:9229'
```
Comment (siberowl) [inline docker-compose.yaml:11]: What is this port for? Is there a reason for choosing 9229?

**Review by siberowl — APPROVED** [review]
This is awesome thanks so much!!

File: README.md (hunk 1 above)
Comment (Njima1572) [reply README.md:34]: From nodejs documentation. Don't have any meaning for me
https://nodejs.org/en/docs/guides/debugging-getting-started/

File: docker-compose.yaml (hunk 2 above)
Comment (Njima1572) [reply docker-compose.yaml:11]: From nodejs documentation. Doesn't have any meaning for me
https://nodejs.org/en/docs/guides/debugging-getting-started/

Comment (siberowl) [reply docker-compose.yaml:11]: I see thanks I was just curious

**Pull request merged**